es6-shim.js

```diff
@@ -1416,41 +1410,55 @@
       return ((al * bl) + (((ah * bl + al * bh) << 16) >>> 0) | 0);
     },
 
-    fround: function fround(x) {
-      var v = Number(x);
-      if (v === 0 || v === Infinity || v === -Infinity || numberIsNaN(v)) {
-        return v;
+    fround: function fround(value) {
+      var x = Number(value);
+      if (x === 0 || x === Infinity || x === -Infinity || Number.isNaN(x)) {
+        return x;
       }
-      var sign = Math.sign(v);
-      var abs = Math.abs(v);
+      var sign = Math.sign(x);
+      var abs = Math.abs(x);
       if (abs < BINARY_32_MIN_VALUE) {
         return sign * roundTiesToEven(abs / BINARY_32_MIN_VALUE / BINARY_32_EPSILON) * BINARY_32_MIN_VALUE * BINARY_32_EPSILON;
       }
       // Veltkamp's splitting (?)
       var a = (1 + BINARY_32_EPSILON / Number.EPSILON) * abs;
       var result = a - (a - abs);
-      if (result > BINARY_32_MAX_VALUE || numberIsNaN(result)) {
+      if (result > BINARY_32_MAX_VALUE || Number.isNaN(result)) {
```

Comment:
For consistency, I'd prefer `numberIsNaN` so that the NaN checks can all be routed through the same code path.